config/routes.rb

```diff
@@ -161,6 +157,19 @@
         patch '/complete' => 'course_choices#complete', as: :course_choices_complete
       end
 
+      scope '/choice/:id' do
+        get '/offer' => 'decisions#offer', as: :offer
```

Comment:
As commented before, this looks like we are retrieving an offer,  
maybe `new_offer_response`,  `new_respond_to_offer`, ...

Reply:
We *are* retrieving an offer here.